docs/source/tutorial.rst

```diff
@@ -144,6 +144,40 @@ The following tools are required:
 * `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_ (5.1.16 r113841
   or later)
 
+
+Proxy Settings
+--------------
+
+If you are behind a network proxy, follow these steps before continuing:
+
+1. Set the following environment variables:
+
+  * http_proxy
+  * https_proxy
+
+If you are using the Bash shell, run the following commands:
+
+.. code-block:: console
+
+  % export http_proxy=http://example-proxy-server.com:911
```

Comment:
OK, I changed to more standard ports, and also put a warning box in to tell user to make sure they are using the actual settings for their environment.